x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -1032,6 +1032,7 @@ export const sendAlertToTimelineAction = async ({
             getExceptionFilter
           );
           // The Query field should remain unpopulated with the suppressed EQL/ES|QL alert.
+          // TODO do we need additional logic for ML alerts here?
```

Comment:
@vitaliidm I wasn't sure what treatment, if any, was necessary here. Do we need to specifically include/exclude ML alerts in the same way as EQL and ES|QL, here? What is necessary in order to support a "suppressed timeline?"

Reply:
Yeah, good catch; looking at this code it doesn't appear that ML alerts are handled in timeline at all :( 

So: I think we need to handle the ML case in general, and then the suppressed ML case specifically. Please let me know if you agree, and I'll try to tackle that tomorrow. I might put that in a subsequent PR, though.

Reply:
For now, I would just add an exclusion for ML rule, similar to EQL/ES|QL.
This way, it would create a general timeline that would show just one alert. 
Similarly to EQL/ES|QL

> So: I think we need to handle the ML case in general, and then the suppressed ML case specifically. Please let me know if you agree, and I'll try to tackle that tomorrow. I might put that in a subsequent PR, though.

How do you see this new implementation of timeline for ML rules? Querying anomaly index?
I think it's out of scope of this PR and worth to discuss with Product team